iothub_client/tests/iothubtr_amqp_tel_msgr_ut/iothubtr_amqp_tel_msgr_ut.c

```diff
@@ -1665,11 +1665,109 @@ static TELEMETRY_MESSENGER_HANDLE create_and_start_messenger2(TELEMETRY_MESSENGE
     return handle;
 }
 
+char* umock_stringify_BINARY_DATA()
```

Comment:
Shouldn't these functions take arguments?

Reply:
Yes, but via our chat, we decided to use the functions as shells for now, I did add the args back in then cast to void, same functionality but at least it looks better

Reply:
Filled in all shell functions in hash 7f188b9